### DISCUSSION TITLE
打嗝也能產出能源

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#928] [General]
https://www.businessweekly.com.tw/carbon-reduction/blog/3010865

文內提及的方法是讓已由為食的微生物投入廢棄油井，讓他們吃飽後排出的氫氣就能拿來使用，但標題為什麼用打嗝這個詞讓我有點疑惑~微生物從古至今與人共存也替人類帶來很多益處，如今又找到了微生物對我們的一項貢獻，不過在它釋放氫氣的過程同時也釋出了二氧化碳，該如何在全球暖化與能源之間取得平衡，我想是全人類共同的問題。

**Comment by peggy0329** [comment]
標題下打嗝的意思應該是讓微生物飽餐一頓後，排放氣體的意思
只是下這個標題不免讓人以為是人類打嗝哈哈哈
我覺得文中善用微生物製造氫氣是好的方式
但研究團隊在無法確定可以購入油井下，就直接授權技術給石油公司
不免讓人為他們捏把冷汗，不怕對方直接把技術挖走嗎

**Comment by Evan0979** [comment]
感覺有點類似沼氣發電的概念!!

**Comment by CPCpyt** [comment]
雖然的確含有能量，不過如何捕捉和運用是一個很大的問題。

**Comment by orianna901001** [comment]
我剛剛看到標題時，頓時想到身邊一整天都在打嗝的朋友，如果真的可以藉此來發電，或許我朋友可以直接去應徵打嗝人發電的職位XD
